README.md

```diff
@@ -113,7 +113,7 @@ See the [`CONTRIBUTING.md`](CONTRIBUTING.md) file for more information.
 OpenThread is released under the [BSD 3-Clause license](LICENSE). See
 the [`LICENSE`](LICENSE) file for more information.
 
-Please only use the OpenThread name and marks when accurately
-referencing this software distribution, and do not use these marks in
+Please use only the OpenThread name and marks when accurately
```

Comment:
This edit changes the meaning of this sentence significantly. I highly recommend reverting the change on this line to the previous wording.